projects/net-snmp/build.sh

```diff
@@ -27,6 +27,18 @@ $CXX $CXXFLAGS $WORK/snmp_pdu_parse_fuzzer.o \
       -Wl,-Bstatic -lcrypto -Wl,-Bdynamic -lm \
       -o $OUT/snmp_pdu_parse_fuzzer
 
+$CC $CFLAGS -c -Iinclude $SRC/snmp_parse_fuzzer.c -o $WORK/snmp_parse_fuzzer.o
+$CXX $CXXFLAGS $WORK/snmp_parse_fuzzer.o \
+      $LIB_FUZZING_ENGINE snmplib/.libs/libnetsnmp.a \
+      -Wl,-Bstatic -lcrypto -Wl,-Bdynamic -lm \
+      -o $OUT/snmp_parse_fuzzer
+
+$CC $CFLAGS -c -Iinclude $SRC/snmp_scoped_pdu_parse_fuzzer.c -o $WORK/snmp_scoped_pdu_parse_fuzzer.o
+$CXX $CXXFLAGS $WORK/snmp_scoped_pdu_parse_fuzzer.o \
+      $LIB_FUZZING_ENGINE snmplib/.libs/libnetsnmp.a \
+      -Wl,-Bstatic -lcrypto -Wl,-Bdynamic -lm \
+      -o $OUT/snmp_scoped_pdu_parse_fuzzer
+
```

Comment:
I see four times the sequence of almost the same $CC and $CXX commands. Please use a for-loop instead of repeating the same two commands four times. Linking all code with both libnetsnmp and libnetsnmpagent is fine.

Reply:
Fixed!